HowToFIDO.md

```diff
@@ -977,3 +955,26 @@ a UVRA</span>](#signing-in-with-a-uvra).
 15. Historically, we used the term "resident key" to refer to what will
     be known as "discoverable credentials" in the upcoming WebAuthn L2
     and CTAP2.1 specs.
+
+
+16.  A note on attestation: Whether or not to specify the attestation 
+     conveyance parameter `attestation`depends on your business priorities.
+     If attestation is not specified, you will not get the AAGUID 
+     (Authenticator Attestation GUID) of the authenticator and you will not
+     know anything about the authenticator you are about to accept for 
+     registration with your services. Knowing AAGUID enables you to find 
+     out the provenance and supported features of the authenticator as well 
+     as if the authenticator is certified by FIDO Alliance from such a source
+     like [FIDO Metadata Service] (https://fidoalliance.org/metadata/). 
+     FIDO Certification assures interoperability as well as levels of security 
+     of authenticators. This will be useful information especially for security 
+     conscious RPs and applications. 
+     If you are not interested in such information of the authenticators at all, 
+     your choice will be not to specify the attestation conveyance parameter 
+     `attestation` (thus defaulting to none), or instead explicitly use the value 
+     `indirect`. This guarantees the most streamlined user experience (platforms 
+     are likely to obtain consent from the user for other types of attestation 
+     conveyances, which likely results in a larger fraction of unsuccessful 
+     credential creations due to users canceling the creation).
+
```

Comment:
I think it makes sense to also note that when asking for attestation there may be some UX concerns (I believe some browsers, at least in some versions, display specific warning when attestation is requested by the browser...?) 
And that if using attestations the RP should be prepared to a case where an authenticator is being used that's not listed in MDS or otherwise known to the RP.